.changeset/strong-pigs-return.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/components": minor
+---
+
+Exported types from LikertScale
```

Comment:
This doesn't seem right 😛

Reply:
It was never exported and Storybook needs it

Reply:
Ohhh. So it's actually not a mistake 🤯